Multisig/UI/Safe Management/Switch Safe/AddSafeRow.swift

```diff
@@ -20,7 +20,7 @@ struct AddSafeRow: View {
             HStack(spacing: 6) {
                 Image.plusCircle
 
-                Text("Add Safe").font(Font.gnoBody.weight(.medium))
+                Text("Add Safe").font(.gnoBody)
```

Comment:
.body() we can define color input parameter

Reply:
I think you already did it right?

Reply:
not using `body()` because otherwise the `foregroundColor` is not working when applied to the whole hstack